application/comit_node/build.rs

```diff
@@ -1,81 +0,0 @@
-extern crate regex;
```

Comment:
👍Feels good

Reply:
@da-kami remove the dependency from `Cargo.toml` please!

Reply:
done